src/Nether.Analytics/MessagePipeline.cs

```diff
@@ -25,7 +26,7 @@ public class MessagePipeline
             _outputManagers = outputManagers;
         }
 
-        public async Task ProcessMessageAsync(Message msg)
+        public async Task ProcessMessageAsync(string partitionId, Message msg)
```

Comment:
Why is there the new partitionId parameter? The Message type that is passed in has a PartitionId property